neo/app/assets/javascripts/lib/adf-structures-base-0.1.1/.jshintrc

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
It's the other dashboard framework I showed you before, not sure if we will use it later, so I keep it.